dev_tools/packaging/publish-dev-package.sh

```diff
@@ -105,6 +107,9 @@ cd "$(git rev-parse --show-toplevel)"
 tmp_package_dir=$(mktemp -d "/tmp/publish-dev-package_package.XXXXXXXXXXXXXXXX")
 trap "{ rm -rf ${tmp_package_dir}; }" EXIT
 
+# Configure to push to cirq-dev and not cirq.
+export CIRQ_DEV_VERSION=`dev_tools/packaging/set-dev-version.sh`
```

Comment:
`export CIRQ_DEV_VERSION=$(dev_tools/packaging/set-dev-version.sh)` to be consistent with the rest of the file

Reply:
done